src/ui/autocomplete-ui-handler.ts

```diff
@@ -1,36 +1,22 @@
+import OptionSelectUiHandler from "#app/ui/option-select-ui-handler";
 import { Button } from "#enums/buttons";
-import AbstractOptionSelectUiHandler from "./abstact-option-select-ui-handler";
-import { Mode } from "./ui";
 
-export default class AutoCompleteUiHandler extends AbstractOptionSelectUiHandler {
-  modalContainer: Phaser.GameObjects.Container;
-  constructor(mode: Mode = Mode.OPTION_SELECT) {
-    super(mode);
-  }
-
-  getWindowWidth(): number {
-    return 64;
-  }
+export default class AutoCompleteUiHandler extends OptionSelectUiHandler {
+  private modalContainer: Phaser.GameObjects.Container;
 
   override show(args: any[]): boolean {
-    if (args[0].modalContainer) {
+    if (args[0].hasOwnProperty("modalContainer")) {
```

Comment:
NIT: please try to avoid using `hasOwnProperty()`.
TS compiler doesn't like it

```suggestion
    if (args[0]?.modalContainer) {
```